bin/millau/node/src/command.rs

```diff
@@ -147,7 +147,10 @@ pub fn run() -> sc_cli::Result<()> {
 			let runner = cli.create_runner(&cli.run)?;
 			runner.run_node_until_exit(|config| async move {
 				match config.role {
-					Role::Light => service::new_light(config),
+					Role::Light => {
```

Comment:
Let's just remove it as it is done in substrate node template: https://github.com/paritytech/substrate/blob/master/bin/node-template/node/src/command.rs#L113